src/iuse.cpp

```diff
@@ -4106,8 +4106,10 @@ int iuse::mp3( player *p, item *it, bool, const tripoint & )
         p->add_msg_if_player( m_info, _( "You put in the earbuds and start listening to music." ) );
         if( it->typeId() == "mp3" ) {
             it->convert( "mp3_on" ).active = true;
+            p->mod_moves( -200 );
         } else if( it->typeId() == "smart_phone" ) {
             it->convert( "smartphone_music" ).active = true;
+            p->mod_moves( -200 );
         }
```

Comment:
Nitpick, but you could replace two lines of `mod_moves`, one for mp3 and one for smartphone, with one, somewhere after line 4105.

Reply:
Due to the lack of any “else” condition in either of these, I though it safest to wait only for known devices.